README.md

```diff
@@ -72,6 +72,34 @@ docker-compose up
 
 Navigate to http://localhost/LEAF_Nexus or http://localhost/LEAF_Request_Portal in your browser.
 
+### Contributing
+
+#### Branches
+
+Major branches:
+
+* `master`: Contains the production ready code.
+  * **NEVER** commit into `master`. **ONLY** pull requests from the `dev` branch and urgent hotfixes can be merged into `master`.
+* `dev`: Contains finished code from the current sprint, ready for merging into `master` at any time.
+  * **NEVER** commit into `dev`, create a pull request.
+
+Typical workflow:
+
+1. Start with the `dev` branch. Ensure `dev` is up to date on your machine.
+1. Create a new feature branch from `dev` with the format `leaf###_short_feature_description`
+    * Each branch name should begin with lowercase `leaf###` where `###` is the issue/ticket number and end with a short description of the feature. 
+1. Do your work. Ensure proper tests are created (where applicable) and all existing tests pass.
+1. Rebase the feature branch into as few logical commits as possible (1-3 total commits is ideal). 
+1. Create a [good commit message](https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message). Keep the commit subject under 50 characters, and wrap the commit message body at 72 characters.
+1. Push feature branch to remote origin.
+1. Teammates will comment and/or approve the changes.
+1. Make any necessary changes.
+1. After review/approval, merge `dev` branch into the feature branch to ensure it works with any other committed feature branches. 
+1. Rebase feature branch (again) into as few logical commits as possible (1-3 total commits is ideal). 
+1. Push merged feature branch to remote.
+1. Create pull request to merge feature branch into `dev`.
```

Comment:
And this should be changed to:

1. The pull request from the feature branch will be automatically updated.